lib/src/drawings.dart

```diff
@@ -96,8 +94,7 @@ class TextDrawing extends GYWDrawing {
 
   @override
   List<GYWBtCommand> toCommands() {
-    final int fontSize = size ?? font?.size ?? GYWFonts.small.font.size;
-    final int charHeight = (fontSize * 1.33).ceil();
+    final int charHeight = (size * 1.33).ceil();
```

Comment:
I don't think I want to put char height as a property in font. This value doesn't make sense when we will add non-monospaced fonts. Besides, why would you want it to make this property part of the public interface?

If, for some reason, you want to give the user monospace properties about fonts that you provide, then lets have 2 types of fonts: monospace and non-monospace (idk if this is what they are called). This way we don't have to deprecate anything when we add non-monospace fonts.

Reply:
I see what you mean, but for now, there are no non-monospaced (idk either if it's the correct name) fonts. So, in the current version, we can assume that all fonts are monospaced.

`charHeight` will probably have meaning for non-monospaced fonts too. Perhaps, it will refer to maximum char height, but we will still need this value to compute wrapping and features like this. I agree that with `charWidth` it is less obvious, but it may also refers to the maximum width of a character. I personally use these value a lot when building a screen with variable size content in order to place an icon at the right location for example.

Reply:
I see. I didn't foresee this use case. Indeed, if you want to compute layout and have other drawings' positions depend on its siblings, than ok, the best we can do right now it to provide a max width and height.

Reply:
What do you think about the latest commit?

1.333 is actually just a constant because it's the size of a pixel per per point.
0.6 (charWidth) can be derived from the aspect ratio of the font. All fonts, monospaced and non-monospaced have an aspect ratio, and ours is 0.45.

Reply:
Let's have 1.333 hardcoded like you did with the calculation, but it shouldn't be in the public API.

For the aspect ratio, how do you make the translation from the aspect ratio to char width ? Is it 0.45 * 1.333 ? I feel like charWidth is easier to understand and to use, but I see the more general idea behind it. In non)monospaced font, the aspect ratio is fixed ?